backend/app/services/mongo/projects.py

```diff
@@ -321,18 +326,20 @@ async def add_project_events(project_id: str, events: List[EventDefinition]) ->
     return updated_project
 
 
-async def email_project_tasks(
+async def email_project_data(
     project_id: str,
     uid: str,
     limit: Optional[int] = 5_000,
-):
+    scope: Literal["tasks", "users"] = "tasks",
+    filters: Optional[ProjectDataFilters] = None,
```

Comment:
shoudln't you pass the filters from the API endpoint as well ? otherwise, not really useful

Reply:
thank you, removed them